smqtk_iqr/web/search_app/modules/iqr/iqr_search.py

```diff
@@ -363,13 +367,13 @@ def set_iqr_session_state():
             # Update service state
             self._iqr_service.put('state',
                                   sid=sid,
-                                  state_base64=service_zip_base64)
+                                  state_base64=base64.b64decode(service_zip_base64).decode('acii'))
```

Comment:
I'm fairly certain that these b64 variables need to be decoded to ascii, but would like confirmation

Reply:
No, these should not be decoded here. This specific call is passing along the base64 payload across the network to another service. The base64 payload is already in ASCII as that is the purpose of base64 encoding.

Reply:
Ok. I had decoded because I was assuming that the requests were expecting strings, but I can see that thats incorrect based on the other comment